.metals/metals.lock.db

```diff
@@ -0,0 +1,6 @@
+#FileLock
+#Thu Dec 03 12:08:40 GMT 2020
+hostName=localhost
+id=1762881221a7123c544904eed0ec4292c00c5eee843
+method=file
+server=localhost\:55841
```

Comment:
I think you may want to add the `.metals` directory to either the project `.gitignore` or your global `.gitignore`.

Reply:
Do we need metals in apps rendering?! do we have any scala code?

Reply:
Is it the Thrift definitions?

Reply:
There's a _tiny_ bit of Scala/SBT code related to [generating the Thrift models](https://github.com/guardian/apps-rendering/tree/main/api-models) for communication between MAPI and AR, as @buck06191 says.